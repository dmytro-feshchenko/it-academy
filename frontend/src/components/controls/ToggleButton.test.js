import React from 'react';
import renderer from 'react-test-renderer';
import ToggleButton from './ToggleButton';

it("renders correctly without checked prop", () => {

    const tree = renderer.create(
        <ToggleButton label="Select" checkedLabel="Deselect" />
    );
    expect(tree).toMatchSnapshot();
});

it("renders correctly with checked prop", () => {

    const tree = renderer.create(
        <ToggleButton label="Select" checkedLabel="Deselect" checked={true} />
    );
    expect(tree).toMatchSnapshot();
});

