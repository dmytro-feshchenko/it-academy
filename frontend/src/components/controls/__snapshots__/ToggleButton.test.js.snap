// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with checked prop 1`] = `
<div
  className="button button_switch button_checked"
  onClick={[Function]}
>
  Deselect
</div>
`;

exports[`renders correctly without checked prop 1`] = `
<div
  className="button button_switch"
  onClick={[Function]}
>
  Select
</div>
`;
